compass/landice/mesh.py

```diff
@@ -529,10 +529,10 @@ def build_cell_width(self, section_name, gridded_dataset,
     geom_points, geom_edges = set_rectangular_geom_points_and_edges(*bnds)
 
     # Remove ice not connected to the ice sheet.
-    flood_mask = gridded_flood_fill(thk) == 0
-    thk[flood_mask] = 0.0
-    vx[flood_mask] = 0.0
-    vy[flood_mask] = 0.0
+    flood_mask = gridded_flood_fill(thk)
+    thk[flood_mask == 0] = 0.0
+    vx[flood_mask == 0] = 0.0
+    vy[flood_mask == 0] = 0.0
```

Comment:
I don't understand why the original change didn't work.  I would think it would be identical to the following, which would be identical to the code after reversion.

``` python
    flood_mask = gridded_flood_fill(thk)
    mask = (flood_mask == 0)
    thk[mask] = 0.0
    vx[mask] = 0.0
    vy[mask] = 0.0
```

But it's not worth exploring further.